pkg/kubelet/kubelet.go

```diff
@@ -3095,6 +3095,8 @@ func (kl *Kubelet) isContainerRuntimeVersionCompatible() error {
 		if result < 0 {
 			return fmt.Errorf("container runtime version is older than %s", dockertools.MinimumDockerAPIVersion)
 		}
+	case "rkt":
+		return kl.GetRuntime().(*rkt.Runtime).CheckVersion(rkt.MinimumRktBinVersion, rkt.RecommendedRktBinVersion, rkt.MinimumAppcVersion, rkt.MinimumRktApiVersion, rkt.MinimumSystemdVersion)
```

Comment:
I think it would be better to implement the `Compare` interface instead of this to convert it to `rkt.Runtime`?
BTW, why don't we just add something like `CheckVersion()` in runtime interface instead, so that we don't have to explicitly "minimum version" here.